lib/domain/notification/usecase/fest_notifications.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:corona_trace/network/api_repository.dart';
+import 'package:corona_trace/network/notification/response_notification.dart';
+
+class FetchNotificationsUseCase {
+  // TODO add DI to handle this for us
+  final ApiRepository _repository = ApiRepository.instance;
+
+  /// Will return a [Future] of type [ResponseNotifications] when called.
+  ///
+  /// Optional [pageIndex] starting at 0 for paginated notification lists.
+  Future<ResponseNotifications> execute({int pageIndex = 0}) async {
+    return _repository.getNotificationsList(pageIndex + 1);
```

Comment:
@matthiasbruns why not sending exact page no ?

Reply:
FetchNotificationsUseCase line 12

Reply:
It feels weird as a caller to not start at 0 when working with indexes.

Reply:
This feels to my like hidden business logic, which should not be handles be the calling parties

Reply:
@matthiasbruns these are no arrays to use the term indexes, these are pages, and should not be 0 .

Reply:
done